.github/ISSUE_TEMPLATE/suggestion.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Suggestion
+about: Suggest something to enhance Skript
+
+---
+
+*Please remove/edit all help text like this before submitting the issue!*
+
+**Description**
+*A clear and concise description of the suggestion*
+
+**Describe the solution you'd like**
+*A clear and concise description of what you want to happen.*
+
+**Describe alternatives you've considered**
```

Comment:
Yeah it got confusing because i removed the first section from the default template:

![image](https://user-images.githubusercontent.com/19474570/45350746-3bb58b80-b5bd-11e8-91f8-39a874df74c2.png)

Because i think that is unnecesary (a `Description` section includes it) and causes confusing when the suggestion is not related to a problem.

So i removed the 2nd and 3th sections, now there are only **Description** and **Additional Context** sections. In this case it looks like the suggestion template is unnecessary. But i also added a **Question* template with just **Question** and **Additional Context** sections just to be more user friendly. So users will see the accepted issue types and it will be better than this:

![image](https://user-images.githubusercontent.com/19474570/45351140-17a67a00-b5be-11e8-944f-e04e1899dbfa.png)

If there is only a `Bug Report` template (which is required), and if the user wants to suggest something, they will have to click on this small *button*. A bit hard to explain but it is better to have separated `Suggestion` and `Question` buttons.

But of course it is still available to create any other types of issues using the *Open a regular issue* button.